geomesa-gs-plugin/geomesa-accumulo-gs-plugin/README.md

```diff
@@ -1,11 +1,33 @@
-# GeoMesa Plugin
+# GeoMesa Accumulo GeoServer Plugin
 
-### Building Instructions
+### Installation Instructions
```

Comment:
mention that if you are connected to the internet and have installed the tools you can use the wget script from `$GEOMESA_HOME/bin/install-hadoop-accumulo.sh` ?